src/start_location.cpp

```diff
@@ -369,6 +369,10 @@ void start_location::place_player( player &u ) const
         }
     }
 
+    int posx = u.posx();
+    int posy = u.posy();
+    g->update_map( posx, posy ); // The player's submap must be in the center of the map
```

Comment:
Can be replaced with just `g->update_map( &u )` (quite similar code inside the function).
Side note: the function should accept references instead of pointers, especially since it's main usage is: `update_map(&...)`